package.json

```diff
@@ -76,5 +77,10 @@
         "preset": "ts-jest"
       }
     ]
+  },
+  "dependencies": {
+    "project": "^0.1.6",
```

Comment:
this isn't installing the dependency you think it is. [it's installing this 9 year old pacakge](https://www.npmjs.com/package/project)